### PR TITLE
Revised Communication Doc: Includes feedback

### PR DESCRIPTION
**Pull request by maluhoss**
Taking in feedback on the first PR by @woozyking  & second PR by @odesiuz . Sections include:
- Introduction
- Communication Tools
- Opt-in/Opt-out
- Choosing Appropriate tools
- Meetings
- Balancing Work with Communication

There are 2 references to newly created files embedded in the document for those looking to get more guidance about certain things like notification management and having a productive meeting. New files don't contain any content at the moment but will be added. Feel free to share your thoughts about this revised document.

### CODE REVIEW

File: communications.md [hunk 1]
```diff
@@ -1,89 +1,49 @@
...
+When there is doubt, use Slack.
 
-Documentation - Transparency: Don't be embarrassed about asking "silly" questions. Try to always write your messages in a publically accessible (or at least a team accessible) channel. This way, the information is preserved and become searchable. Loosely, it becomes a part of the knowledge base, aside from more formal documentation.
+## Meetings
 
-A few exceptions:
+If a meeting is necessary, we have meeting spaces that can be pre-booked and used appropriately. Please be mindful of the duration time and goal for the meeting as we presently have limited private space for meetings and others might be waiting in line. If you’d like to get tips on how to ensure a productive meeting, click [here](https://github.com/EQWorks/common/productive-meeting.md).
```
Comment (YizhiCatherineZhang) [inline communications.md]: Shall we mention the meeting agenda part? Personally, I think to have a meeting agenda before the meeting will help people understand whether they should attend this meeting or what should prepare for this meeting to save some time.

File: communications.md [hunk 2]
```diff
@@ -1,89 +1,49 @@
...
 
-Budget communication time. Try to allocate yourself a relatively fixed schedule to initiate, check and respond to messages. To a certain extent, this means try to optimize for the whole block of contiguous time that suits better for your job (knowledge-based, creative endeavors).
+Slack is our primary tool for communication where one can message, ping or ask questions to team members in channels depending on projects, interests and teams e.g. #snoke #random #general
 
-#### - Internal & External Messaging Tool: Slack -
-Opt-in: Join channels that you are interested in or obligated to.
+G Suite consists of 1) Gmail, used for client and team correspondence and 2) Calendar, used for setting up meetings by creating an event invitation & booking one of the available rooms.
 
-Setting up Notifications: Once you're part of a channel, set up your notifications to suit your needs:
+Asana is commonly used for project management and any other processes we deem it fit to be used for (e.g Locus Builder Redesign, Office Availability, Weekly Meetings, etc.)
```
Comment (YizhiCatherineZhang) [inline communications.md]: Shall we also mention the knowledge base part? If it's not an urgent question, they can ask there and we can do weekly clean up.

File: communications.md (hunk 1 above)
Comment (maluhoss) [reply communications.md]: great point. I was initially going to put that in productive meeting link but that makes sense. will add that.

File: communications.md (hunk 2 above)
Comment (maluhoss) [reply communications.md]: yes good point. will add that in as another process we use for Asana. Perhaps we can even have a specific section about where information can be found (if one has a question) and mention that one can either search in Slack or go to Asana project

**Pull request closed without merging**